ElvUI_OptionsUI/Search.lua

```diff
@@ -0,0 +1,227 @@
+local E, _, V, P, G = unpack(ElvUI)
+local C, L = unpack(select(2, ...))
+local ACH = E.Libs.ACH
+local SearchText = ''
+
+local gsub = gsub
+local wipe = wipe
+local next = next
+local type = type
+local pcall = pcall
+local pairs = pairs
+local ipairs = ipairs
+local gmatch = gmatch
+local tinsert = tinsert
+local strtrim = strtrim
+local strfind = strfind
+local strjoin = strjoin
+local strlower = strlower
+local strmatch = strmatch
+local strsplit = strsplit
+
+local start = 100
+local depth = start + 2
+local inline = depth - 1
+local results, entries = {}, {}
+local sep = ' |cFF888888>|r '
+
+local searchCache = {}
+local blockOption = {
+	filters = true,
+	info = true,
+	plugins = true,
+	search = true,
+	tagGroup = true,
+	modulecontrol = true,
+	profiles = true
+}
+
+local typeInvalid = {
+	description = true,
+	header = true
+}
+
+local typeValue = {
+	multiselect = true,
+	select = true,
+}
+
+local nameIndex = {
+	[L["General"]] = 1,
+	[L["Global"]] = 0
+}
+
+E.Options.args.search = ACH:Group(L["Search"], nil, 4)
+local Search =  E.Options.args.search.args
+
+local EditBox = ACH:Input(L["Search"], nil, 0, nil, 1.5, function() return SearchText end, function(_, value) C:Search_ClearResults() if strmatch(value, '%S+') then SearchText = strtrim(strlower(value)) C:Search_Config() C:Search_AddResults() end end)
+-- Search.editbox = EditBox
+
+E.Options.args.searchEditBox = EditBox
+E.Options.args.searchEditBox.order = 2
+
+local Header = {
+	order = 1,
+	type = "header",
+	name = "Пишем что хочем найти слево сверху в ту штуку где написанно поиск и кликаем Enter\n Результаты будут отображены в развертке вкладки поиск слево",
+	width = "full"
+}
+local Header2 = {
+	order = 2,
+	type = "header",
+	name = "Результаты будут отображены в развертке вкладки поиск слево",
```

Comment:
Is this second header necessary? Text seems to be about the same as the text after `\n` in the previous header.

`name = "The results will be shown under the search category on the left",`